Assets/Mona/MonaLibrary/MonaLibraryWindow.cs

```diff
@@ -0,0 +1,316 @@
+#if UNITY_EDITOR
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using Unity.EditorCoroutines.Editor;
+using UnityEngine.Networking;
+
+namespace Mona
+{
+    public class MonaLibraryWindow : EditorWindow
+    {
+        static MonaLibraryWindow window = null;
+        private readonly string at_PublicAPIKey = "keywC7DhH4fzXGWcg";
+        private readonly string at_BaseID = "appglbIlOT8JLLnur";
+        private readonly string at_RootTableID = "tblf3EHvQM36tPLJE"; 
```

Comment:
Since these are readonly, can we use a naming convention similar to constants: `AIRTABLE_PUBLIC_API_KEY`